src/main/java/com/epam/ta/reportportal/auth/token/extractor/decorator/ExcludedPathTokenExtractor.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.ta.reportportal.auth.token.extractor.decorator;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.provider.authentication.TokenExtractor;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+
+public class ExcludedPathTokenExtractor implements TokenExtractor {
```

Comment:
Could you add test for this class please.

Reply:
Done